zcash_client_memory/src/mem_wallet/wallet_read.rs

```diff
@@ -210,11 +210,27 @@ impl WalletRead for MemoryWalletDb {
         &self,
         _min_confirmations: u32,
     ) -> Result<Option<WalletSummary<Self::AccountId>>, Self::Error> {
+        // let summary = WalletSummary::new(
+        //     account_balances,
+        //     chain_tip_height,
+        //     fully_scanned_height,
+        //     None,
+        //     next_sapling_subtree_index,
+        //     #[cfg(feature = "orchard")]
+        //     next_orchard_subtree_index,
+        // );
+        // Ok(Some(summary))
         todo!()
     }
 
     fn chain_height(&self) -> Result<Option<BlockHeight>, Self::Error> {
-        todo!()
+        Ok(self
```

Comment:
I'm a little unsure of this is the right way to do this. Not certain more of a feeling